app/home/homepage.tsx

```diff
@@ -57,32 +48,22 @@ export default function Homepage({
                 </div>
             </div>
 
-            <DataSection runs={runs} gamestats={gamestats} />
-        </div>
-    );
-}
-
-const DataSection = ({
-    runs,
-    gamestats,
-}: {
-    runs: Run[];
-    gamestats: Game[];
-}) => {
-    return (
-        <div>
-            <Row className="text-center">
-                <Col xl={6} lg={12} className="mt-4">
-                    <h2>Recent Personal Bests</h2>
-                    {runs && <DataHolder runs={runs} />}
-                    {!runs && <SkeletonPersonalBests />}
-                </Col>
-                <Col xl={6} lg={12} className="mt-4">
-                    <h2>Popular Games</h2>
-                    {gamestats && <PopularGames gamestats={gamestats} />}
-                    {!gamestats && <SkeletonPopularGames />}
-                </Col>
-            </Row>
+            <div>
+                <Row className="text-center">
+                    <Col xl={6} lg={12} className="mt-4">
```

Comment:
```suggestion
                    <Col xl={6} className="mt-4">
```
`lg={12}` can be removed completely, it's always col-12 in this case